v63004/outref/gtm8900.txt

```diff
@@ -0,0 +1,29 @@
+# Create a single region DB with region DEFAULT
+# Check the DB
+
+# Testing MUPIP SET -[NO]ENCRYPT in a properly set up environment
+# --------------------------------------------------------------------------
+# \##SOURCE_PATH##/mupip set -ENCRYPTABLE -REGION 'DEFAULT'
+Database file ##TEST_PATH##/mumps.dat now has encryptable flag set to  TRUE
+
+# \##SOURCE_PATH##/mupip set -NOENCRYPTABLE -REGION 'DEFAULT'
+Database file ##TEST_PATH##/mumps.dat now has encryptable flag set to FALSE
+
+
+# Testing MUPIP SET -ENCRYPT  with no set gtm_passwd (expecting failure)
+# --------------------------------------------------------------------------
+# \##SOURCE_PATH##/mupip set -ENCRYPTABLE -REGION 'DEFAULT'
+%YDB-E-CRYPTINIT2, Could not initialize encryption library during GT.M startup. Environment variable ydb_passwd/gtm_passwd not set
+
+# \##SOURCE_PATH##/mupip set -NOENCRYPTABLE -REGION 'DEFAULT'
+Database file ##TEST_PATH##/mumps.dat now has encryptable flag set to FALSE
+
+
+# Testing MUPIP SET -ENCRYPT  with garbage GNUPGHOME (expecting failure)
+# --------------------------------------------------------------------------
+GNUPGHOME: garbageValue
+# \##SOURCE_PATH##/mupip set -ENCRYPTABLE -REGION 'DEFAULT'
+%YDB-E-CRYPTINIT2, Could not initialize encryption library during GT.M startup. Cannot stat on garbageValue - 2
+
+# \##SOURCE_PATH##/mupip set -NOENCRYPTABLE -REGION 'DEFAULT'
```

Comment:
Additionally, I want tests for MUPIP SET -NOENCRYPT where the database had encryptable flag set to TRUE but GNUPGHOME is garbage or not (randomly chosen) and gtm_passwd is not set or set (randomly chosen). We expect success in all these cases.